### ISSUE TITLE
Conflicting models

### ISSUE THREAD

**Issue by wontonst** [treyhunner/django-simple-history#345]
In my application, I have an `Inventory` class and a `HistoricalInventory` class. Hence, when I try to use simple history on Inventory, I get

```
RuntimeError: Conflicting 'historicalinventory' models in application 'main': <class 'myapp.main.HistoricalInventory'> and <class 'myapp.main.models.HistoricalInventory'>.
```

The preexisting `HistoricalInventory` is used in many places so refactoring it would be a pain. Do you think it makes sense to make the model name a configurable field, eg

`history = HistoricalRecords(historical_model_name='SimpleHistoricalInventory')`

If so, I can quickly put up a PR and unblock myself.

**Comment by rossmechanic** [comment]
Happy to accept a PR that does with with the necessary tests.

**Comment by rossmechanic** [comment]
Closed by #451
